app/views/students/show.html

```diff
@@ -0,0 +1,125 @@
+<div ng-controller='StudentsShowCtrl as controller'>
+	<!-- Page Heading -->
+	<div class="row">
+		<div class="col-xs-12">
+			<h2 class="page-header">
+        <span translate="i18nStudentsPanelSection"></span>
```

Comment:
I'm not even sure why it's there. It should the the student's firstName + lastName.